Integrations/integration-GoogleResourceManager.yml

```diff
@@ -0,0 +1,1020 @@
+category: IT Services
+commonfields: {id: Google Resource Manager, version: -1}
+configuration:
+- defaultvalue: ''
+  display: The Project ID of the project resource in which you created the Service
+    Account. The value can be found in the Service Account Key File which should have
+    downloaded when creating the Service Account in the Google Cloud Platform.
+  name: project_id
+  options: []
+  required: true
+  type: 0
+- defaultvalue: ''
+  display: Value of the 'private_key_id' field in your JSON Service Account Key File.
+  name: private_key_id
+  options: []
+  required: true
+  type: 4
+- defaultvalue: ''
+  display: Value of the 'private_key' field in your JSON Service Account Key File.
+  name: private_key
+  options: []
+  required: true
+  type: 4
+- defaultvalue: ''
+  display: Value of the 'client_email' field in your JSON Service Account Key File.
+  name: client_email
+  options: []
+  required: true
+  type: 0
+- defaultvalue: ''
+  display: Value of the 'client_id' field in your JSON Service Account Key File.
+  name: client_id
+  options: []
+  required: true
+  type: 4
+- defaultvalue: ''
+  display: Value of the 'client_x509_cert_url' field in your JSON Service Account
+    Key File.
+  name: client_x509_cert_url
+  options: []
+  required: true
+  type: 0
+description: Google Cloud Platform Resource Manager
+detaileddescription: Use these detailed instructions in order to retrieve the API
+  key
+display: Google Resource Manager
+name: Google Resource Manager
+script:
+  commands:
+  - arguments:
+    - default: false
+      defaultValue: ''
+      description: |-
+        The unique, user-assigned ID of the Project. It must be 6 to 30 lowercase letters, digits, or hyphens. It must start with a letter. Trailing hyphens are prohibited.
+        Example: tokyo-rain-123
+      isArray: false
+      name: projectId
+      predefined: ['']
+      required: true
+    - default: false
+      defaultValue: ''
+      description: |-
+        string
+        The id of the parent resource.
+      isArray: false
+      name: parent_id
+      predefined: []
+      required: true
+    - default: false
+      defaultValue: ''
+      description: |-
+        The label keys associated with this Project.
+        Label keys must be between 1 and 63 characters long and must conform to the following regular expression: [a-z]([-a-z0-9]*[a-z0-9])?.
+        No more than 256 labels can be associated with a given resource.
+      isArray: true
+      name: labels.attrs
```

Comment:
improve argument name, don't spare on characters and don't use dots, but underscores